app/src/main/java/org/mozilla/fenix/tabstray/SyncedTabsInteractor.kt

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray
+
+import mozilla.components.browser.storage.sync.Tab
+
+/**
+ * Interactor to be called for any actions on synced tabs in the tabs tray.
+ */
+interface SyncedTabsInteractor {
+    /**
+     * Invoked when the user clicks on a SyncedTab item.
```

Comment:
```suggestion
     * Invoked when the user clicks on a synced [Tab] item.
```
Maybe add an @param while we are at it, but the point was to make sure we are correcting the docs since we don't have a `SyncedTab`(?).